<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>XGUI</class>
 <widget class="QMainWindow" name="XGUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>565</width>
    <height>431</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::DefaultContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>Display Settings</string>
  </property>
  <property name="windowIcon">
   <iconset resource="XGUI.qrc">
    <normaloff>:/video-display.png</normaloff>:/video-display.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_4">
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="pixmapLabel1">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Plain</enum>
        </property>
        <property name="pixmap">
         <pixmap resource="XGUI.qrc">:/monitor.png</pixmap>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
        <property name="alignment">
         <set>Qt::AlignTop</set>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="Line" name="line2">
        <property name="frameShape">
         <enum>QFrame::VLine</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Sunken</enum>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>5</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item row="0" column="1">
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0">
       <spacer name="spacer11">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>330</width>
          <height>16</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="1" column="0">
       <widget class="QTabWidget" name="tabWidget">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="tab_2">
         <attribute name="title">
          <string>General</string>
         </attribute>
         <layout class="QGridLayout" name="gridLayout_5">
          <item row="0" column="0">
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <widget class="QLabel" name="label_5">
              <property name="text">
               <string>Video Card:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="labelVideoCard">
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="pixmapLabel1_6">
              <property name="minimumSize">
               <size>
                <width>32</width>
                <height>32</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>32</width>
                <height>32</height>
               </size>
              </property>
              <property name="pixmap">
               <pixmap resource="XGUI.qrc">:/kcontrol.png</pixmap>
              </property>
              <property name="scaledContents">
               <bool>true</bool>
              </property>
              <property name="wordWrap">
               <bool>false</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="1" column="0">
           <layout class="QHBoxLayout" name="horizontalLayout_5">
            <item>
             <widget class="QLabel" name="label_6">
              <property name="text">
               <string>Monitor:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="labelMonitor">
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="pixmapLabel1_7">
              <property name="minimumSize">
               <size>
                <width>32</width>
                <height>32</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>32</width>
                <height>32</height>
               </size>
              </property>
              <property name="pixmap">
               <pixmap resource="XGUI.qrc">:/video-display.png</pixmap>
              </property>
              <property name="scaledContents">
               <bool>true</bool>
              </property>
              <property name="wordWrap">
               <bool>false</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="2" column="0">
           <layout class="QHBoxLayout" name="_9">
            <item>
             <widget class="QLabel" name="textLabel1_5">
              <property name="text">
               <string>Screen Resolution:</string>
              </property>
              <property name="wordWrap">
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="comboResolution">
              <item>
               <property name="text">
                <string>1024x768</string>
               </property>
              </item>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="pixmapLabel1_3">
              <property name="minimumSize">
               <size>
                <width>32</width>
                <height>32</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>32</width>
                <height>32</height>
               </size>
              </property>
              <property name="pixmap">
               <pixmap resource="XGUI.qrc">:/randr.png</pixmap>
              </property>
              <property name="scaledContents">
               <bool>true</bool>
              </property>
              <property name="wordWrap">
               <bool>false</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="3" column="0">
           <layout class="QHBoxLayout" name="_10">
            <item>
             <widget class="QLabel" name="textLabel1_6">
              <property name="text">
               <string>Color Depth:</string>
              </property>
              <property name="wordWrap">
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="comboDepth">
              <item>
               <property name="text">
                <string>24bits</string>
               </property>
              </item>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="pixmapLabel1_2_4">
              <property name="minimumSize">
               <size>
                <width>32</width>
                <height>32</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>32</width>
                <height>32</height>
               </size>
              </property>
              <property name="pixmap">
               <pixmap resource="XGUI.qrc">:/colors.png</pixmap>
              </property>
              <property name="scaledContents">
               <bool>true</bool>
              </property>
              <property name="wordWrap">
               <bool>false</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="4" column="0">
           <layout class="QHBoxLayout" name="_11">
            <item>
             <widget class="QLabel" name="textLabel1_7">
              <property name="text">
               <string>Video Driver</string>
              </property>
              <property name="wordWrap">
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="comboDriver"/>
            </item>
            <item>
             <widget class="QLabel" name="pixmapLabel1_2_7">
              <property name="minimumSize">
               <size>
                <width>32</width>
                <height>32</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>32</width>
                <height>32</height>
               </size>
              </property>
              <property name="pixmap">
               <pixmap resource="XGUI.qrc">:/kcontrol.png</pixmap>
              </property>
              <property name="scaledContents">
               <bool>true</bool>
              </property>
              <property name="wordWrap">
               <bool>false</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="6" column="0">
           <spacer name="spacer2_2">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Preferred</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>16</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="5" column="0">
           <widget class="QLabel" name="labelRecDriver">
            <property name="text">
             <string/>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
            <property name="wordWrap">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="TabPage_2">
         <attribute name="title">
          <string>Advanced</string>
         </attribute>
         <layout class="QGridLayout" name="gridLayout_6">
          <item row="0" column="0">
           <widget class="QCheckBox" name="checkMonitorSync">
            <property name="text">
             <string>Monitor Settings</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QCheckBox" name="checkDualHead">
            <property name="text">
             <string>Enable Dual-Head support</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QGroupBox" name="groupDualHead">
            <property name="title">
             <string/>
            </property>
            <layout class="QGridLayout" name="gridLayout_2">
             <item row="0" column="0">
              <layout class="QHBoxLayout" name="horizontalLayout_6">
               <item>
                <widget class="QLabel" name="label_7">
                 <property name="text">
                  <string>Monitor Resolution:</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QComboBox" name="comboDualRes"/>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="label_8">
            <property name="text">
             <string>Dual-Head support is designed for systems with one video card and two monitors. Other setups may be achieved by editing /etc/X11/xorg.conf</string>
            </property>
            <property name="wordWrap">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QGroupBox" name="groupMonitor">
            <property name="title">
             <string/>
            </property>
            <layout class="QGridLayout" name="gridLayout_8">
             <item row="0" column="0">
              <layout class="QHBoxLayout" name="_13">
               <item>
                <widget class="QLabel" name="textLabel1_8">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="text">
                  <string>Horizontal Sync:</string>
                 </property>
                 <property name="wordWrap">
                  <bool>false</bool>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="lineHorz">
                 <property name="text">
                  <string>28.0-78.0</string>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignHCenter</set>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="pixmapLabel1_2_8">
                 <property name="minimumSize">
                  <size>
                   <width>32</width>
                   <height>32</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>32</width>
                   <height>32</height>
                  </size>
                 </property>
                 <property name="pixmap">
                  <pixmap resource="XGUI.qrc">:/smserver.png</pixmap>
                 </property>
                 <property name="scaledContents">
                  <bool>true</bool>
                 </property>
                 <property name="wordWrap">
                  <bool>false</bool>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item row="1" column="0">
              <layout class="QHBoxLayout" name="_14">
               <item>
                <widget class="QLabel" name="textLabel1_4_3">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="text">
                  <string>Vertical Refresh:</string>
                 </property>
                 <property name="wordWrap">
                  <bool>false</bool>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="lineVertRefresh">
                 <property name="text">
                  <string>50.0-76.0</string>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignHCenter</set>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="pixmapLabel1_2_6_3">
                 <property name="minimumSize">
                  <size>
                   <width>32</width>
                   <height>32</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>32</width>
                   <height>32</height>
                  </size>
                 </property>
                 <property name="pixmap">
                  <pixmap resource="XGUI.qrc">:/smserver.png</pixmap>
                 </property>
                 <property name="scaledContents">
                  <bool>true</bool>
                 </property>
                 <property name="wordWrap">
                  <bool>false</bool>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
      <item row="2" column="0">
       <spacer name="spacer12">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Preferred</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>327</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item row="0" column="2">
     <spacer name="horizontalSpacer_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Fixed</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>10</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="1" column="0" colspan="2">
     <widget class="Line" name="line1">
      <property name="frameShape">
       <enum>QFrame::HLine</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Sunken</enum>
      </property>
     </widget>
    </item>
    <item row="2" column="0" colspan="2">
     <spacer name="spacer5_2">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Preferred</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>530</width>
        <height>13</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="3" column="0" colspan="2">
     <layout class="QHBoxLayout" name="_8">
      <property name="margin">
       <number>0</number>
      </property>
      <item>
       <spacer name="spacer1_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::MinimumExpanding</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>311</width>
          <height>21</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="pushApply">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Apply</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushClose">
        <property name="text">
         <string>Skip</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="spacer4_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>21</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="XGUI.qrc"/>
 </resources>
 <connections/>
</ui>
